upup/models/cloudup/resources/addons/nodelocaldns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -0,0 +1,176 @@
+---
```

Comment:
Can you include a commented link to where this manifest was copied from?

Reply:
Sure, make sense